modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectEntry/ModalDeleteObjectEntry.tsx

```diff
@@ -0,0 +1,102 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import ClayButton from '@clayui/button';
+import ClayModal from '@clayui/modal';
+import {fetch, openToast} from 'frontend-js-web';
+import React from 'react';
+
+interface ItemData {
+	actions: ItemDataDelete;
+	id: number;
+}
+
+interface ItemDataDeleteProps {
+	href: string;
+	method: string;
+}
+
+interface ModalDeleteObjectEntry {
```

Comment:
@Artartur Please, add "Props" at the end of this interface's name. You don't need to do it for the other interfaces, just this one because it's the main interface used for the default component in this file.